src/components/NavigationTree/__stories__/NavigationTree.stories.tsx

```diff
@@ -1,55 +1,77 @@
 import React from 'react';
 
 import {Button} from '@gravity-ui/uikit';
-import type {Meta, Story} from '@storybook/react';
+import type {Meta, StoryFn} from '@storybook/react';
 
 import {NavigationTree} from '../NavigationTree';
 import type {NavigationTreeProps} from '../NavigationTree';
 import type {NavigationTreeDataItem, NavigationTreeNodeType} from '../types';
 
 export default {
+    parameters: {
+        controls: {expanded: true},
+    },
     title: 'NavigationTree',
     component: NavigationTree,
+    args: {
+        cache: true,
+        virtualize: false,
+        rootState: {
+            path: '',
+            name: 'ru/maps/maps_prod',
+            type: 'database',
+            collapsed: false,
+        },
+        getActions: getActions,
+    },
+    argTypes: {
+        rootState: {
+            control: false,
+        },
+        fetchPath: {
+            control: false,
+        },
+        getActions: {
+            control: false,
+        },
+        renderAdditionalNodeElements: {
+            control: false,
+        },
+        activePath: {
+            control: false,
+        },
+        onActivePathUpdate: {
+            control: false,
+        },
```

Comment:
I only allowed changing `cache` and `virtualize`, other props are disabled as they were before. The reason - other props are complex and may require some setup, that I do not want to do right now